.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!-- Please take a look at our [Contributing](https://rook.io/docs/rook/master/development-flow.html)
+documentation before submitting a Pull Request!
+Thank you for contributing to Rook! -->
+
+Is this a fix for an issue or a feature implementation?
+<!-- Remove only one -->
+* Fix
+* Feature
+
+Description of your changes:
+
+Which issue is resolved by this Pull Request:
+
+Does your change break an existing feature:
```

Comment:
This should be part of the description above (implicit), and will be covered by the checklist below. So can we remove this separate line?